core/frontend/package.json

```diff
@@ -39,30 +39,30 @@
     "url": "http://www.bentley.com"
   },
   "peerDependencies": {
+    "@bentley/itwin-client": "workspace:^3.0.0-dev.122",
+    "@itwin/appui-abstract": "workspace:^3.0.0-dev.122",
     "@itwin/core-bentley": "workspace:^3.0.0-dev.122",
-    "@itwin/core-geometry": "workspace:^3.0.0-dev.122",
     "@itwin/core-common": "workspace:^3.0.0-dev.122",
-    "@itwin/core-quantity": "workspace:^3.0.0-dev.122",
-    "@bentley/itwin-client": "workspace:^3.0.0-dev.122",
+    "@itwin/core-geometry": "workspace:^3.0.0-dev.122",
     "@itwin/core-orbitgt": "workspace:^3.0.0-dev.122",
-    "@itwin/appui-abstract": "workspace:^3.0.0-dev.122",
+    "@itwin/core-quantity": "workspace:^3.0.0-dev.122",
```

Comment:
I think we should remove these carets. Recent NPM versions install peer-dependencies automatically and they usually pick up a version that doesn't work.

Reply:
I think once we release a new pre-release like `-rc` it should resolve correctly in those builds. The extension prereleases we did kinda screws it up at the moment

Reply:
actually I talked to Naveed and he had a problem where his project was using -dev.72 but NPM kept pulling in -dev.113 for peer-dependencies. I don't think an RC will fix this, because there's nothing preventing NPM from pulling in a more recent RC build (assuming we have multiple RCs before final release like usual) that could be incompatible in some internal API.

Reply:
Part of the problem is that we want, I think, the real release versions to use the `^` and how do we differentiate between the two.

Reply:
But do we? What if someone uses a 3.0 app and npm pulls in 3.2 peer-dependencies, can we guarantee it will work?

Reply:
_If_ we're following semver, then pulling in 3.2 **shouldn't** break apps on 3.0; can we guarantee that, not sure :)

Reply:
yeah that's true, but also have `@internal`/`@alpha`/`@beta` APIs that don't necessarily follow semver rules. That is the only part I'm worried about, because we do use those between our own packages.

Reply:
Yes, I think you're right @paulius-valiunas. This is honestly how we end up using it within all of our apps, we explicitly set each of the versions.

I'll possibly follow up with a separate PR to do that where needed.